lib/serverspec/type/curl_command.rb

```diff
@@ -0,0 +1,48 @@
+module Serverspec
+  module Type
+    class CurlCommand < Command
+      def response_code
+        m = /Response-Code: (?<code>\d+)/.match(stderr)
```

Comment:
This is incompatible with Ruby 1.8.7. Is that still something that should still be supported?